### DISCUSSION TITLE
Subtitles colour

### DISCUSSION THREAD

**Discussion by JoseteLuisete** [smplayer-dev/smplayer#308] [Q&A]
Hi!
With the following configuration ...
Options > Preferences > Subtitles > Font and colors:
- Use custom style: Yes
- Text: White
- Border: Black

The text is in dark gray. I want the text in white.
Any ideas?
If I set "Use custom style = No" the text is still in dark gray.
Thanks in advance!

**Comment by JoseteLuisete** [comment]
Anobody?
I have just found out that this happens only in HEVC + 4K + MKV videos.
It doesn't happen in my AVC or HEVC 1080p videos.
Is this a bug?

**Reply by smplayer-dev** [reply]
Does it make any difference if you select another video output in preferences -> general -> video?

**Comment by JoseteLuisete** [comment]
You are right, this happens only with "xv"

Then in these kinds of videos (MKV+4K+HEVC) it seems I see problems with all the video outputs ...

vaapi : Very dull colors
xv : Grey subtitles and dull colors (solved activating Equalizer > Saturation +25 & Hue -15)
Default : Jumpy playback
gpu : Jumpy playback
vdpau : Jumpy playback
wlshm : Jumpy playback
drm : Jumpy playback

 Greetings!
Jan 4, 2022, 15:41 by ***@***.***:
